pyrasterframes/src/main/python/setup.py

```diff
@@ -89,9 +139,8 @@ def run(self):
     install_requires=requirements,
```

Comment:
See this [article](https://packaging.python.org/discussions/install-requires-vs-requirements/) for a discussion relevant to this practice. Though it seems like a violation of DRY to define `install_requires` in setup.py and a separate requirments.txt; they do express different purposes.

Reply:
uhg

Reply:
I think this is well reasoned discussion also https://caremad.io/posts/2013/07/setup-vs-requirement/